docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -7,33 +7,48 @@ To deploy this framework in your environment, follow the steps outlined below.
 
 #### 1. Unzip the release bundle
 
-#### 2. Navigate to the Dev Tools console in Kibana
+#### 2. Modify artifacts to reflect Kibana space
+For security reasons, we recommend that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `include_space.py` script with your Kibana space as the argument.
 
-#### 3. Uploading required scripts
+Eg:
 
-Upload the scripts in `ml_hostriskscore_levels_script.json`, `ml_hostriskscore_map_script.json` and `ml_hostriskscore_reduce_script.json` using the following API call:
+```
+python include_space.py --space default
```

Comment:
Should we call this `generate_scripts.py` or something more declarative like that? I now see the JSON files in `dreml` as templates, so in my opinion "scripts" don't really exist until you run this python script to generate them.

Reply:
Done!